pkg/metrics/metrics.jsx

```diff
@@ -887,7 +887,7 @@ class CurrentMetrics extends React.Component {
                                     value={memTotal ? this.state.memUsed : undefined}
                                     className="pf-m-sm"
                                     min={0} max={memTotal}
-                                    variant={memUsedFraction > 0.9 ? ProgressVariant.danger : null}
+                                    variant={memUsedFraction > 0.9 ? ProgressVariant.danger : memUsedFraction >= 0.8 ? ProgressVariant.warning : null}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19217-20230822-113626-06b76265-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)